apps/web/src/components/HyloEditor/HyloEditorMobile.js

```diff
@@ -101,7 +100,7 @@ export default function HyloEditorMobile (props) {
     className: 'hyloEditorMobile',
     onUpdate: handleUpdate,
     onEnter: handleEnter,
-    onBeforeCreate: handleBeforeCreate,
+    onCreate: handleCreate,
```

Comment:
This is the key change, and only impacts HyloEditorMobile. Previously, likely because of different behaviour in an earlier version of TipTap, we handled setting the "loaded" state, which is the only thing we use this event for, via `onBeforeCreate`. That was too early now, so now hooking into `onCreate` to set editor loaded which HyloEditorMobile pays attention to to know when it can send editor updates through the WebView.